packages/flutter/test/material/animated_icons_private_test.dart

```diff
@@ -104,7 +104,7 @@ void main () {
       painter.paint(mockCanvas, size);
       expect(generatedPaths.length, 1);
 
-      verifyInOrder(<dynamic>[
+      verifyInOrder(<void>[
```

Comment:
I think you will need to pass a type argument to verifyInOrder to satisfy their lints.

```
verifyInOrder<void>(<void>[...
```